### ISSUE TITLE
Map broken since last update

### ISSUE THREAD

**Issue by francobam** [War3zuk/War3zuk-Alpha-18-Stable-AIO#2]
Hey there,

since your last update today the Blunt map is broken. POI are hanging in the air, no trees and the biomes look mixed. My base is still in a good shape, but everything around is weird and disappears on getting closer.

Cheers
Frank

**Comment by War3zuk** [comment]
Yeah I know, Your not on my Discord to have seen the Fiasco I went through 2 days ago were my Repo went shits up... Had to remake a new repo & sort out all the files again having to remake each map... Files are all ok now.. Also when making the new maps i took the chance to update them along with being able to add the extra files so it runs abit better on lower end pcs.. A FULL fresh install is needed & yes im well aware its broken everything as I had to also wipe my own server..
